source/cli-implementation.js

```diff
@@ -106,6 +110,7 @@ try {
 		tests: true,
 		publish: true,
 		releaseDraft: true,
+		releaseNotes: true,
```

Comment:
Maybe this could default to false. The "Generate release notes" button which now exists tends to work a little better for me.